services/ui-src/src/verbiage/pages/mcpar/mcpar-review-and-submit.ts

```diff
@@ -1,6 +1,16 @@
 export default {
   intro: {
-    header: "MCPAR Review and Submit",
-    body: "Placeholder page",
+    header: "Review & Submit",
+    infoHeader: "Ready to Submit?",
+    info: "Double check that everything in your MCPAR Report is accurate. You will be able to make edits after submitting, and resubmit. Once you’ve reviewed your report, certify that it’s in compliance with 42 CFR § 438.66(e).",
+  },
+  modal: {
+    action: "Submit MCPAR",
+    body: "You will be able to make edits to this MCPAR after submitting, and resubmit.",
+    close: "Cancel",
+    heading: "Are you sure you want to submit MCPAR?",
```

Comment:
Nit: Order these so that they flow like the modal and maybe update the names for clarity, like:
```suggestion
    heading: "Are you sure you want to submit MCPAR?",
    body: "You will be able to make edits to this MCPAR after submitting, and resubmit.",
    actionButtonText: "Submit MCPAR",
    closeButtonText: "Cancel",
    
```